package.json

```diff
@@ -1,7 +1,7 @@
 {
   "scripts": {
     "functionalTest": "mocha --require ./tests/functional/setupVpn.js --timeout 30000",
-    "functionalTestWasm": "mocha --require ./tests/functional/setupWasm.js --timeout 30000",
+    "functionalTestWasm": "WASM=true mocha --require ./tests/functional/setupWasm.js --timeout 30000",
```

Comment:
Right now we set `this.currentText.ctx.wasm = true` before each test case, which can be useful to verfy if a test is running on the WASM environment. 

In this PR I introduce a whole test suite that I want to skip on WASM, so that state doesn't help much because the check needs to happen before the `beforeEach` block in run. Also we need some state that is accessible from outside a test case. Using an environment variable was the way I found to do this.